packages/ag-charts-community/src/chart/caption.test.ts

```diff
@@ -0,0 +1,109 @@
+import { afterEach, beforeEach, describe, expect, jest, test } from '@jest/globals';
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+
+import type { TextAlign } from '../options/agChartOptions';
+import type { Chart } from './chart';
+import {
+    IMAGE_SNAPSHOT_DEFAULTS,
+    createChart,
+    extractImageData,
+    setupMockCanvas,
+    waitForChartStability,
+} from './test/utils';
+
+expect.extend({ toMatchImageSnapshot });
+
+describe('PieSeries', () => {
```

Comment:
Nit: `'Caption'`?

Reply:
Well spotted! I've changed this string. I'll merge once it's green.